subsys/net/lib/lwm2m_client_utils/lwm2m/lwm2m_lte_notification.c

```diff
@@ -17,6 +17,10 @@
 
 #include <zephyr/logging/log.h>
 LOG_MODULE_REGISTER(lwm2m_neighbour_cell, CONFIG_LWM2M_CLIENT_UTILS_LOG_LEVEL);
```

Comment:
Maybe change the module name as the module isn't just about ncell anymore.

Reply:
true I can do it.